public/main.js

```diff
@@ -0,0 +1,497 @@
+let playerArray = [];
+let turnCount = 0;
+let trueCount = 0;
+let blockArray = [];
+let currentBox;
+let playerXName = 'X';
+let playerOName = 'O';
+let playerName;
+let player;
+let currentPlayer = 'X';
+let timer;
+let count = 6;
+let noClick = [];
+let index;
+let onePlayer = false;
+let compMove;
+
+let boxes = Array.from(document.getElementsByClassName("box"));  //array of all divs to make setting event listeners easier
+
+class Box {  //each box is an object with an element to point to, a value, and a clicked boolean
+    constructor(element, value) {
+        this.element = element;
+        this.value = value;
+        this.clicked = false;
+        this.xClick = false;
+        this.oClick = false;
+    }
+}
+let one = new Box(document.getElementById("one"), 1);
+let two = new Box(document.getElementById("two"), 2);
+let three = new Box(document.getElementById("three"), 3);
+let four = new Box(document.getElementById("four"), 4);
+let five = new Box(document.getElementById("five"), 5);
+let six = new Box(document.getElementById("six"), 6);
+let seven = new Box(document.getElementById("seven"), 7);
+let eight = new Box(document.getElementById("eight"), 8);
+let nine = new Box(document.getElementById("nine"), 9);
```

Comment:
This is a bit more complicated than it needs to be .  All the properties you are setting in your `Box` class are available from the html element itself. You've already got an array of all the html elements, you don't need to make new objects to represent them. If you really want to set it up this way you could still condense it by using a `forEach` loop on your `boxes` array. You don't need to set each box up individually.